src/features/auth/Login.tsx

```diff
@@ -258,7 +259,7 @@ export default function Login({
                 onIonChange={(e) => setServer(e.target.value)}
               >
                 <IonList inset>
-                  {servers.slice(0, 3).map((server) => (
```

Comment:
The limit for up to 3 is if a self hosted instance has a lot of custom servers, so we should probably keep it.

Reply:
Wouldn't it be better in that case, since a self hosted instance user can easily select their domain when they log in?

Would the list not scroll if there are too many items on the list? I haven't checked that case :thinking: